src/Components/App.jsx

```diff
@@ -12,25 +13,27 @@ class App extends React.Component {
 
   componentDidMount() {
     const setState = (value) => {
-      this.setState({data: value});
+      this.setState({data: value });
     }

```

Comment:
I see what you're doing here, but you should probably change the name of your function to something else to avoid confusion with this.setState. Perhaps something like "setStateOfDataKey"?